kernelci/cli.py

```diff
@@ -165,6 +165,11 @@ class Args:
         "Path to the install directory, or _install_ inside kdir by default",
     }
 
+    is_admin = {
+        'name': '--is-admin',
+        'help': "Provide 1 for superuser and 0 for regular user",
```

Comment:
Add this to make it a boolean flag:
```python
        'action': 'store_true'
```
Then the `--is-admin` option can be used with no argument.

Reply:
Done.